pulpcore/pulpcore/app/models/repository.py

```diff
@@ -221,25 +207,147 @@ class RepositoryContent(Model):
 
         content (models.ForeignKey): The associated content.
         repository (models.ForeignKey): The associated repository.
+        vadded (models.ForeignKey): The RepositoryVersion which added the referenced Content.
+        vremoved (models.ForeignKey): The RepositoryVersion which removed the referenced Content.
     """
-    content = models.ForeignKey('Content', on_delete=models.CASCADE)
+    content = models.ForeignKey('Content', on_delete=models.CASCADE,
+                                related_name='version_memberships')
     repository = models.ForeignKey(Repository, on_delete=models.CASCADE)
+    vadded = models.ForeignKey('RepositoryVersion', related_name='added_memberships')
+    vremoved = models.ForeignKey('RepositoryVersion', null=True, related_name='removed_memberships')
```

Comment:
I don't get the *memberships* theme in `related_name`.  Can you help me understand the thinking here.

Reply:
Yeah, maybe another word would be better. The idea is that on a repo version object, there will be a django manager attribute that corresponds to all of the ``RepositoryVersion`` objects that add something for that version. What should that attribute be named? ``added_repository_contents`` seems awkward. Thinking of these objects as "memberships" seemed like a reasonable noun to replace "repository_contents". But introducing a new word might just make things worse.

What do you think it should be called? Perhaps ``added_repository_contents`` isn't so bad?